### PR TITLE
WF-63458-XlsIOMAUI-HF

### PR DESCRIPTION
**Pull request by KonduruKeerthi**
Hi All,

Kindy review the changes and let me know your feedback.

Regards,
Keerthi

### CODE REVIEW

File: File-Formats/XlsIO/Create-read-edit-excel-files-in-maui-c-sharp.md [hunk 1]
```diff
@@ -242,85 +242,96 @@ using (ExcelEngine excelEngine = new ExcelEngine())
...
-    DependencyService.Get<ISave>().SaveAndView("Invoice.xlsx", "application/excel", ms);
+    SaveService saveService = new SaveService();
+    saveService.SaveAndView("Output.xlsx", "application/vnd.openxmlformats-officedocument.spreadsheetml.sheet", ms);
 }
 {% endhighlight %}
 {% endtabs %}  
 
+## Save Service class in portable project
+
+Add the new class file with name as **SaveService** to the **Project** and add below code in it.
```
Comment (johnsonmonohar) [inline File-Formats/XlsIO/Create-read-edit-excel-files-in-maui-c-sharp.md:255]: Please mention the usage of this class.

**Review by johnsonmonohar — CHANGES_REQUESTED** [review]
Please address the comment.

File: File-Formats/XlsIO/Create-read-edit-excel-files-in-maui-c-sharp.md (hunk 1 above)
Comment (KonduruKeerthi) [reply File-Formats/XlsIO/Create-read-edit-excel-files-in-maui-c-sharp.md]: I have mentioned the usage as suggested.

**Review by johnsonmonohar — APPROVED** [review]
Approved.

**Pull request merged**